execution_engine_testing/tests/src/test/regression/mod.rs

```diff
@@ -24,7 +25,6 @@ mod ee_550;
 mod ee_572;
 mod ee_584;
 mod ee_597;
-mod ee_598;
```

Comment:
Note that we've disabled the EE-598 regression for the moment, as it uses proxy contract in a way that no longer works with the current checks.  

A fix will follow...